ct/graylog.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
+source <(curl -s https://raw.githubusercontent.com/community-scripts/ProxmoxVE/main/misc/build.func)
+# Copyright (c) 2021-2025 community-scripts ORG
+# Author: Slaviša Arežina (tremor021)
+# License: MIT | https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+# Source: https://graylog.org/
+
+APP="Graylog"
+TAGS="logging"
+var_cpu="2"
+var_ram="8192"
+var_disk="30"
+var_os="debian"
+var_version="12"
+var_unprivileged="1"
+
+header_info "$APP"
+base_settings
+
+variables
+color
+catch_errors
+
+function update_script() {
+    header_info
+    check_container_storage
+    check_container_resources
+
+        msg_info "Updating $APP"
+
+        msg_info "Stopping $APP"
+        systemctl stop graylog-datanode
+        systemctl stop graylog-server
+        msg_ok "Stopped $APP"
+
+        msg_info "Updating $APP"
+        apt-get update &> /dev/null
+        apt-get upgrade -y &> /dev/null
+        msg_ok "Updated $APP"
+
+        msg_info "Starting $APP"
+        systemctl start graylog-datanode
+        systemctl start graylog-server
+        msg_ok "Started $APP"
+
+        msg_ok "Update Successful"
```

Comment:
Fix spacing and add check if the update script is run in Graylog lxc
```suggestion
    header_info
    check_container_storage
    check_container_resources

    if [[ ! -d /etc/graylog ]]; then
        msg_error "No ${APP} Installation Found!"
        exit
    fi
    
    msg_info "Updating $APP"

    msg_info "Stopping $APP"
    systemctl stop graylog-datanode
    systemctl stop graylog-server
    msg_ok "Stopped $APP"

    msg_info "Updating $APP"
    apt-get update &> /dev/null
    apt-get upgrade -y &> /dev/null
    msg_ok "Updated $APP"

    msg_info "Starting $APP"
    systemctl start graylog-datanode
    systemctl start graylog-server
    msg_ok "Started $APP"

    msg_ok "Update Successful"
```